### DISCUSSION TITLE
What will be the config options for following usage?

### DISCUSSION THREAD

**Discussion by bheeshmpita** [crimx/ext-saladict#1917] [Q&A] [answered]
I am not able to select the options for my usage. Please help me in doing so. 
When i double-click the word an icon appears alongside the selection and when i hover the mouse over it and it should reveal the definitions. How can i achieve this?
I have gone through tutorial on website before submitting this question.

翻译文本
我无法为我的使用选择选项。请帮助我这样做。
当我双击单词时，选择旁边会出现一个图标，当我将鼠标悬停在它上面时，它应该会显示定义。我怎样才能做到这一点？
在提交这个问题之前，我已经在网站上浏览了教程。

**Comment by crimx** [comment, marked as answer]
Hi! You mean you only want the icon to appear only on double-click, not just normal selection, right? Sorry there is no option for it.

**Comment by crimx** [comment]
You can find all the available options under "Search Modes" tab.

**Reply by bheeshmpita** [reply]
Thanks for a quick reply. 
In search modes- normal selection-"Hover on the bowl icon to trigger searching instead of clicking." what is this this option for if my original query doesnt have such a mode for that?

**Reply by crimx** [reply]
You can choose to either hover or click on the icon to trigger searching.
